ansible/templates/.env_default.j2

```diff
@@ -10,6 +10,4 @@ vault_pki=true
 
 # Default variables for hashistack
 VAULT_ADDR=http://127.0.0.1:8200
```

Comment:
Should we also get rid of this?

Reply:
That's good questions. @fredrikhgrelland WDYT?

Reply:
The vault addr we can keep. Vault defaults to https, so we must have that variable to use vault

Reply:
@fredrikhgrelland : Can't we explicitly set the VAULT_ADDR when and where needed in ansible files, similar to how we are setting VAULT_TOKEN and CONSUL_HTTP_TOKEN ?
As per my understanding, **VAULT_ADDR=http://127.0.0.1:8200** is box specific just like **VAULT_TOKEN=master** is, so I wonder why we should treat these two differently!

Reply:
Yes, we could, but I would argue this *is* different. the token we want you, as a user of the box or the module, to have an understanding of, but **where** vault is is a configuration setting that will be out of scope when using the module.